src/SignalR/clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -650,6 +651,45 @@ export class TransportSendQueue {
     }
 }
 
+class UnsupportedTransportError extends Error {
```

Comment:
Error classes should go in https://github.com/dotnet/aspnetcore/blob/main/src/SignalR/clients/ts/signalr/src/Errors.ts